docs/source/configuration.rst

```diff
@@ -312,8 +312,8 @@ BLAST configuration options
 
 .. _elb_blast_db_margin:
 
-``BLAST database memory margin`` 
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+``BLAST database memory margin (NOT currently supported)`` 
```

Comment:
I would remove this parameter from documentation for now, because I do not see it implemented any time soon.

Reply:
Removed